components/ModDashboardLeftNavbar.vue

```diff
@@ -38,19 +38,19 @@ const modDashboardViewOptions = [{
 const submissionListViewOptions = [{
```

Comment:
nit: unrelated to this, but I think the UI elements would be easier if they were static vs a dynamic list. 
Then you can just import the icons with the nice named component style. 

for this PR, this is fine though

Reply:
@NabbeunNabi Could you make a ticket for this suggestion?